podcasts/Analytics/Adapters/Newform AppsFlyer/AppsFlyerAdapter.swift

```diff
@@ -48,11 +59,13 @@ class AppsFlyerAdapter: AnalyticsAdapter {
         }
         let shouldShowPrompt = appTrackingTransparencyProvider.shouldShowPrompt()
         if !shouldShowPrompt, appTrackingTransparencyProvider.userGaveConsent() {
+            FacebookCore.Settings.shared.isAdvertiserTrackingEnabled = true
```

Comment:
This can be initially added into `appsFlyerSetup` as `FacebookCore.Settings.shared.isAdvertiserTrackingEnabled = appTrackingTransparencyProvider.userGaveConsent()`, but following the documentation, I would do this behind macro for iOS < 17 as
```
Warning: Setter for isAdvertiserTrackingEnabled is deprecated: The setAdvertiserTrackingEnabled flag is not used for FBSDK v17+ on iOS 17+ as the FBSDK v17+ now relies on ATTrackingManager.trackingAuthorizationStatus.
```
As we also refresh this when the auth is called, maybe it's better to wrap this into a func?